### PR TITLE
Added a simple spinner in the connection menu.

### PR DESCRIPTION
**Pull request by PawBud**
Screenshot from 2020-06-29 20-36-08

### CODE REVIEW

File: companion-app/src/PhoneApps/Apps/Hello.jsx [hunk 1]
```diff
@@ -2,15 +2,15 @@ import React from 'react';
 import App from '../App';
 
 import { Typography } from 'antd';
-import {PlayCircleOutlined} from '@ant-design/icons';
```
Comment (arthuro555) [inline companion-app/src/PhoneApps/Apps/Hello.jsx:5]: Please keep that for another PR (or don't do it at all as this is a test app not an app for the end release of the app :wink:)

Comment (PawBud) [reply companion-app/src/PhoneApps/Apps/Hello.jsx:5]: sorry about that, let me force push the change
